app/diagnostics.go

```diff
@@ -241,34 +247,37 @@ func trackConfig() {
 	})
 
 	SendDiagnostic(TRACK_CONFIG_FILE, map[string]interface{}{
-		"enable_public_links": utils.Cfg.FileSettings.EnablePublicLink,
-		"driver_name":         utils.Cfg.FileSettings.DriverName,
-		"amazon_s3_ssl":       *utils.Cfg.FileSettings.AmazonS3SSL,
-		"amazon_s3_signv2":    *utils.Cfg.FileSettings.AmazonS3SignV2,
-		"thumbnail_width":     utils.Cfg.FileSettings.ThumbnailWidth,
-		"thumbnail_height":    utils.Cfg.FileSettings.ThumbnailHeight,
-		"preview_width":       utils.Cfg.FileSettings.PreviewWidth,
-		"preview_height":      utils.Cfg.FileSettings.PreviewHeight,
-		"profile_width":       utils.Cfg.FileSettings.ProfileWidth,
-		"profile_height":      utils.Cfg.FileSettings.ProfileHeight,
-		"max_file_size":       *utils.Cfg.FileSettings.MaxFileSize,
+		"enable_public_links":     utils.Cfg.FileSettings.EnablePublicLink,
+		"driver_name":             utils.Cfg.FileSettings.DriverName,
+		"amazon_s3_ssl":           *utils.Cfg.FileSettings.AmazonS3SSL,
+		"amazon_s3_signv2":        *utils.Cfg.FileSettings.AmazonS3SignV2,
```

Comment:
Discussed with @jasonblais on pre-release. Kept the typo version as we were already tracking with the typo and we don't want a change of key.